docs/content/utilities/moose_docs/docs_standards/docs_markdown_template.md

```diff
@@ -0,0 +1,51 @@
+# Code Class Name (with Spaces)
+`!syntax description /ClassType/CodeClassName`
+
+## Description
+Use the `CodeClassName` as it appears within the code and in the input file within the first paragraph under the description.
+
+Citations for source materials should be made in text with the MooseDocs bibtex extension: \cite{example_reference} (`\cite{example_reference}`) renders as both the author name and the publication year.
+
+If referencing an equation within text, use the abbreviation Eq \eqref{eq:example_equation} (`Eq` `\eqref{eq:example_equation}`).
```

Comment:
It looks like 'Figure' and 'Table' get spelled out automatically by the system. If we're not going to abbreviate them, my opinion is that 'Equation' should also be spelled out for consistency.

Reply:
Do you want \eqref to automatically put in the correct word as well?

Reply:
That makes sense to me.  I'll make that change as I work on the stub page suggestion @aeslaughter  made.

Reply:
Yeah, that would be good.  I assume it could make the word and the number all in the hyperlink text to be consistent with the figures and tables too.

Reply:
@bwspenc Yes, that would be the what I do. I will make that change after @sapitts finishes this up.

Reply:
@aeslaughter I'll get back to this commit tomorrow